card/mod/edit/set/all/edit_in_place.rb

```diff
@@ -1,7 +1,7 @@
 format :html do
   view :edit_in_place, perms: :update, tags: :unknown_ok, cache: :never, wrap: :slot do
     with_nest_mode :edit do
-      card_form :update, success: { view: :editable } do
+      card_form :update do
```

Comment:
this needs to return to the home view.

Reply:
but you have to make sure that the home view has the edit-in-place link

Reply:
what? why?  If the home view didn't have a link to here, then you couldn't get here in the first place, right?

Reply:
You can check out how editing works on metrics/companies on projects now to see that it's important to return to the home view.

Reply:
Before we introduced the edit option the editable view was the only way to get the in-place editor. I'm not questioning the change here. I'm only saying since this became much more involved (in the background) it should be tested that it works. I know that it should work but I have trust issues.

Reply:
Fair point. I'll write some tests.

Reply:
(I did test it in the browser, fwiw, but we should definitely have some specs)